From d268a9d3be66a9ec9114e20b4d608505a4ec3493 Mon Sep 17 00:00:00 2001
From: Johannes Kauffmann <johanneskauffmann@hotmail.com>
Date: Wed, 28 Sep 2022 12:52:41 +0200
Subject: [PATCH 139/157] ANGLE: Fix compilation on GCC 11

GCC 11 no longer transitively includes <limits>, needed for
std::numeric_limits<T>::max and friends, and thus compilation breaks.

Backport from angle upstream at 1b908d0a7ec791fdb856eff56f6bd2da9f93759e.

Edited: the Qt5 version of angle doesn't have the <functional> include.
---
 src/3rdparty/angle/src/libANGLE/HandleAllocator.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/3rdparty/angle/src/libANGLE/HandleAllocator.cpp b/src/3rdparty/angle/src/libANGLE/HandleAllocator.cpp
index c3c184258f..32af3f8f29 100644
--- a/src/3rdparty/angle/src/libANGLE/HandleAllocator.cpp
+++ b/src/3rdparty/angle/src/libANGLE/HandleAllocator.cpp
@@ -10,6 +10,7 @@
 #include "libANGLE/HandleAllocator.h"
 
 #include <algorithm>
+#include <limits>
 
 #include "common/debug.h"
 
-- 
2.38.1

